shared/src/main/scala/mouse/anyf.scala

```diff
@@ -8,4 +12,11 @@ trait AnyFSyntax {
 final class AnyFOps[F[_], A](private val fa: F[A]) extends AnyVal {
   @inline def ||>[G[_]](f: F ~> G): G[A] = f(fa)
   @inline def thrushK[G[_]](f: F ~> G): G[A] = f(fa)
+
+  def liftEitherT[E](implicit F: Functor[F]): EitherT[F, E, A] =
+    EitherT.right[E](fa)
+
+  def liftOptionT(implicit F: Functor[F]): OptionT[F, A] =
```

Comment:
This method could be replaced with `fa.thrushK(OptionT.liftK)` where `thrushK` is from `AnyFOps`.

Reply:
Of course, one could also write `OptionT.liftF(fa)`, but that's the whole point of this MR (and whole `mouse` project IMHO), to make API more user-friendly.

Reply:
My view is that since monad transformers are part of Cats, syntactic convenience for using them is in scope for Mouse. 

(Im away camping ATM, and will review the PR properly when I'm back in the city tomorrow.)